hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -645,7 +648,9 @@ private static void createTableLayoutOnStorage(StorageConfiguration<?> storageCo
     }
 
     initializeBootstrapDirsIfNotExists(basePath, storage);
-    HoodieTableConfig.create(storage, metaPathDir, props);
+    if (shouldCreateTableConfig) {
```

Comment:
shouldn't we always create the table properties file?

Reply:
No, I just need to create the timeline/hstory folder without updating the configs.